aptos-move/block-executor/src/executor.rs

```diff
@@ -1206,7 +1247,39 @@ where
                 executor_arguments,
                 signature_verified_block,
                 base_view,
-            )
+                dynamic_change_set_optimizations_enabled,
+            );
+
+            if ret.is_err() && dynamic_change_set_optimizations_enabled {
+                if let Err(Error::FallbackToSequential(e)) = ret {
+                    match e {
+                        PanicOr::Or(IntentionalFallbackToSequential::ModulePathReadWrite) => {
+                            panic!("ModulePathReadWrite shouldn't happen in sequential execution")
+                        },
+                        PanicOr::Or(IntentionalFallbackToSequential::DirectWriteSetTransaction) => {
+                            info!("[Execution]: DirectWriteSetTransaction found, during ModulePathReadWrite sequential fallback");
+                        },
+                        PanicOr::CodeInvariantError(msg) => {
+                            error!("[Execution]: CodeInvariantError({:?}) in sequential with dynamic_change_set_optimizations_enabled, sequential fallback", msg);
+                        },
```

Comment:
Let's hope that these errors don't occur the second time we run the transactions sequentially. In case they ever occur in production, what's the plan to solve those during on-call quickly?

Reply:
We can disable AGGREGATOR_V2_EXECUTION and corresponding RG flag